job-service/src/main/java/com/hpe/caf/services/job/api/JobsPut.java

```diff
@@ -236,17 +209,33 @@ public static String createOrUpdateJob(final String partitionId, String jobId, N
             throw e;
         }
     }
-
-    private static void closeQueueConnection(final QueueServices queueServices)
+    
+    /**
+     * We trigger the scheduler so it will pick up the created job from the database
+     * then send a message to the appropriate queue
+     */
+    private static void triggerScheduler()
     {
-        try {
-            queueServices.close();
-        } catch (final Exception e) {
-            LOG.warn("Error on connection close to RabbitMQ", e);
+        try (final QueueServices queueServices = QueueServicesFactory.create(config, config.getSchedulerQueue(), codec)){
+            
+            LOG.debug("createOrUpdateJob: Triggering scheduler to send data to the target queue");
+            
+            final TaskMessage taskMessage = new TaskMessage(
+                    UUID.randomUUID().toString(),
+                    DocumentWorkerConstants.DOCUMENT_TASK_NAME,
+                    1,
+                    OBJECT_MAPPER.writeValueAsBytes(new DocumentWorkerDocumentTask()),
+                    TaskStatus.NEW_TASK,
+                    Collections.emptyMap(),
+                    config.getSchedulerQueue());
+            final byte[] taskMessageBytes = serializingData(taskMessage);
+            queueServices.publishMessage(taskMessageBytes);
+        } catch (final Exception ex) {
+           LOG.error("fail to ping the scheduler {}", ex.getMessage());
         }
     }
-
-    private static byte[] getTaskDataBytes(final WorkerAction workerAction, final Codec codec)
+    
+    private static byte[] getTaskDataBytes(final WorkerAction workerAction)
```

Comment:
Were the changes to this method necessary?

Reply:
Aiming to reduce the number of argument by making the codec global. Restoring